lib/svg_sprite/svg_sprite.rb

```diff
@@ -347,4 +340,12 @@ def self.process(icon_name)
     FA_ICON_MAP.each { |k, v| icon_name.sub!(k, v) }
     fa4_to_fa5_names[icon_name] || icon_name
   end
+
+  def self.get_set_cache(key, &blk)
+    cache[key] ||= blk.call
```

Comment:
If you're not doing anything funky with the block, I'd suggest to use `yield` instead and omit the `&blk` param.

Reply:
Done, thanks for the tip